cl/scrapers/tasks.py

```diff
@@ -228,6 +231,30 @@ def get_page_count(path, extension):
     return None
 
 
+def update_document_from_text(opinion):
+    """
+    Given opinion object, generate Juriscraper site object and
+    extract metadata from text (currently only implemented in Tax
+
+    :param opinion: Opinion Object
+    :return:
+    """
```

Comment:
Missing paren, but also, this should have a one line summary and then if needed a multi-line description after a line return. 

Notionally:

```suggestion
    """Extract additional metadata from document text
    
    Use functions from Juriscraper to pull metadata out of opinion
    text. Currently only implemented in Tax, but functional in all 
    scrapers via AbstractSite object.

    :param opinion: Opinion object
    :return: None
    """
```

Reply:
It could be failing because the latest juriscraper master has not yet been pushed.  The two are being updated side by side.

Reply:
Fixed and updated